package.json

```diff
@@ -16,7 +16,7 @@
     "@actions/core": "^1.6.0",
     "@actions/exec": "^1.1.1",
     "@actions/github": "^5.0.1",
-    "@adobe/aio-lib-template-validation": "^2.1.0",
```

Comment:
I'll update this dependency once https://github.com/adobe/aio-lib-template-validation/pull/7 got the second approval.